pkg/apis/infinispan/v1/types_util.go

```diff
@@ -492,3 +517,13 @@ func applyLabels(ispn *Infinispan, envvar, annotationName string) error {
 func (ispn *Infinispan) HasCustomLibraries() bool {
 	return ispn.Spec.Dependencies != nil && ispn.Spec.Dependencies.VolumeClaimName != ""
 }
+
+// HasServiceMonitor returns true by default for ServiceMonitor
+func (ispn *Infinispan) HasServiceMonitor() bool {
```

Comment:
`IsServiceMonitorEnabled`?

Reply:
Yes, thanks, didn't find better name, but this could be fine. Will change